gfauto/gfautotests/test_signature_util.py

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 from pathlib import Path
+
```

Comment:
Deliberate?

Reply:
Yeah actually this is correct, but not currently picked up by the linter.